.buildkite/pipeline.yml

```diff
@@ -106,31 +105,35 @@ steps:
       - label: "GPU runtests"
         command: "julia --color=yes --project=test test/runtests.jl"
         agents:
-          slurm_mem_per_cpu: 8G
-          slurm_gpus: 1
+          partition: gpu
+          slurm_ntasks: 1
+	  slurm_gres: "gpu:1"
 
       - label: "Global Bucket on GPU (functional albedo)"
         key: "global_bucket_function_gpu"
         command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_function.jl"
         agents:
-          slurm_mem: 8G
-          slurm_gpus: 1
+          partition: gpu
+          slurm_ntasks: 1
+	  slurm_gres: "gpu:1"
         artifact_paths: "experiments/standalone/Bucket/artifacts/*gpu*"
 
       - label: "Global Bucket on GPU (static map albedo)"
         key: "global_bucket_staticmap_gpu"
         command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket_staticmap.jl"
         agents:
-          slurm_mem: 8G
-          slurm_gpus: 1
+          partition: gpu
```

Comment:
You no longer need to specify the partition

Reply:
ok! I was following this: https://github.com/CliMA/SurfaceFluxes.jl/blob/c1c7df657b97555058920b178e224b8a0578a593/.buildkite/pipeline.yml#L21

do I keep the partition: expansion line still?

Reply:
No, neither. It doesn't hurt having them, but they no longer required.